### PR TITLE
thin-provisioning-tools: new, 0.9.0

### PR DESCRIPTION
**Pull request by RedL0tus**
Topic Description
-----------------

Introduce thin-provisioning-tools, which is required for thin-provisioning-related functionalities of LVM2.

Package(s) Affected
-------------------

thin-provisioning-tools: new, 0.9.0

Security Update?
----------------

No

Architectural Progress
----------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by RedL0tus** [comment]
@cthbleachbit Kould you pwease review and test dis senpai? UwU

File: extra-admin/thin-provisioning-tools/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,9 @@
+autoconf
```
Comment (MingcongBai) [inline extra-admin/thin-provisioning-tools/autobuild/build]: Use more `abinfo` throughout`, and add verbosity.

File: extra-admin/thin-provisioning-tools/autobuild/build [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+./configure ${AUTOTOOLS_DEF} ${AUTOTOOLS_AFTER}
```
Comment (MingcongBai) [inline extra-admin/thin-provisioning-tools/autobuild/build]: Use absolute path here.

File: extra-admin/thin-provisioning-tools/autobuild/build [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+make
+make DESTDIR="${PKGDIR}" BINDIR="${PKGDIR}"/usr/bin MANDIR=/usr/share/man install
+
```
Comment (MingcongBai) [inline extra-admin/thin-provisioning-tools/autobuild/build]: So... Why did you not use an `ABTYPE`?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**